src/db/initDb.mjs

```diff
@@ -24,4 +24,19 @@ export async function createNodeDbTables(knex) {
       t.jsonb('state');
     });
   }
+
+  const hasInteractionTable = await knex.schema.hasTable('interactions');
+  if (!hasInteractionTable) {
+    await knex.schema.createTable('interactions', function (t) {
+      t.string('id').index().unique();
+      t.string('contract_tx_id').index();
+      t.string('owner_address').index();
+      t.integer('block_height');
+      t.string('tag_index_0').index();
```

Comment:
I think that using such a naive implementation with narrowing to 5 indexes per interaction will most effective and performant than separating interactions and tags too different table for example and them joining them. But I'm not really experienced in SQL so maybe I'm missing something

Reply:
that's fine, KISS.